docs/orchestration/global-hooks/guides.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Global Hooks Guide
+sidebarTitle: "Guide"
+---
+<img
+  alt="Penguin Fishing"
+  src="https://github.com/mage-ai/assets/blob/main/penguin_fishing.gif?raw=True"
+/>
+Follow this step by step guide on creating a global hook in Mage.
+
+### <b>Step 1: Navigate to the Global Hooks UI</b>
+1. Log in to your Mage account.
+1. From the left-hand sidebar, click on "Global Hooks" under the "Settings" section.
```

Comment:
If it is a brand new user, they may need to enable the "Global hooks" feature under their workspace preferences before the Global hooks section actually appears in the left nav, so we should clarify that step.

Reply:
"From the left-hand sidebar, click on "Global Hooks" under the "Settings" section." seems a bit unclear to me since going to the Settings page just lets you enable the feature but not navigate to it, maybe you meant to "click on Global hooks in the left navigation pane"?